staging/src/k8s.io/api/scheduling/v1/types.go

```diff
@@ -49,6 +49,11 @@ type PriorityClass struct {
 	// when this priority class should be used.
 	// +optional
 	Description string `json:"description,omitempty" protobuf:"bytes,4,opt,name=description"`
+
+	// Preempting specifies whether a pod with this PriorityClass could trigger a preemption process.
+	// If this field is unset, the PriorityClass is considered a Preempting class by default.
+	// +optional
+	Preempting *bool `json:"preempting,omitempty" protobuf:"bytes,5,opt,name=preempting"`
```

Comment:
this (or a `preemptionPolicy` field) is a new field that controls PriorityClass behavior. ValidatePriorityClassUpdate today disallows changing value. Should we also disallow changing the preemption policy from changing in ValidatePriorityClassUpdate?

Reply:
Done